docs/limitations.md

```diff
@@ -0,0 +1,296 @@
+# Clear Containers known differences and limitations
+
+As Intel® Clear Containers utilises Virtual Machines (VM) to enhance
+security and isolation of container workloads, the `cc-runtime` has a
+number of differences and limitations when compared with the standard
+Docker runtime, `runc`. Some of these limitations have potential
+solutions, whereas others exist due to fundamental architectural
+differences generally related to the use of VMs.
+
+The Clear Container runtime launches each container within its own
+hardware isolated VM, and each VM has its own kernel. Due to this higher
+degree of isolation, there are certain container capabilities that
+cannot be supported or are implicitly enabled via the VM.
+
+The below sections describe in brief the known limitations, and where
+applicable further links off to the relevant open Issues or pages with
+more detailed information.
+
+## Pending items
+
+This section lists items that may technically be fixable:
+
+### Networking
+
+#### Adding networks dynamically
+
+The runtime does not currently support adding networks to an already
+running container (`docker network connect`).
+
+The VM network configuration is set up with what is defined by the CNM
+plugin at startup time. It would be possible to watch the networking
+namespace on the host to discover and propagate new networks at runtime
+but, it is not implemented today.
+
+See `cc-oci-runtime` issue [\#388](https://github.com/01org/cc-oci-runtime/issues/388).
+
+### Resource management
+
+Due to the way VMs differ in their CPU and memory allocation and sharing
+across the host system, the implementation of an equivalent method for
+these commands is potentially challenging.
+
+#### `docker run -m`
+
+The `docker run -m MEMORY` option is not currently supported. At the
+runtime level, this equates to the `linux.resources.memory` OCI
+configuration. It should be feasible to pass the relevant information
+through to the QEMU `-m` memory size option. This is also related to the
+`docker update` command.
+
+See issue #381 for more information.
```

Comment:
missed link to issue to make the document easy to read and to relate issues faster.